docs/data/sql_connectors.yml

```diff
@@ -169,22 +169,26 @@ upsert-kafka:
 kinesis:
     name: Amazon Kinesis Data Streams
     category: connector
-    maven: flink-connector-kinesis
-    sql_url: https://repo.maven.apache.org/maven2/org/apache/flink/flink-sql-connector-kinesis/$version/flink-sql-connector-kinesis-$version.jar
+    versions:
+        - versions: universal
```

Comment:
This version is meant to be the destination version. Since KDS/KDF/DDB are versionless I have changed to `universal` inline with Kafka. There is background work on improving the shortcodes to deal with external connectors more graefully